docs/Reference/Responsible-Disclosure.md

```diff
@@ -7,7 +7,7 @@ description: Ethsigner responsible disclosure statement.
 At PegaSys, security is a priority. But regardless of how much effort we put into system security,
 there may still be vulnerabilities present.
 If you discover a vulnerability, we want to know about it so we can take steps to address it as
-quickly as possible. We would like to ask you to help us better protect our clients and our systems.
+quickly as possible. You can help us better protect our clients and our systems.
```

Comment:
This file is in all of our docs repos, so we'll need to make this same change in each one.